apps/course/src/app/shared/feedback/loading-bars/loading-bars.component.ts

```diff
@@ -2,13 +2,19 @@ import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'app-loading-bars',
-  standalone: true,
-  imports: [],
   templateUrl: './loading-bars.component.html',
-  styleUrl: './loading-bars.component.scss',
+  styleUrls: ['./loading-bars.component.scss'],
+  standalone: true,
 })
 export class LoadingBarsComponent {
   @Input() size: string = 'lg';
   @Input() marginTop: string = '50';
   @Input() marginBottom: string = '50';
+  @Input() loadingText: string = '';
```

Comment:
Please use type:
```ts
@Input() loadingText: string | null;
```
And in the template check if `loadingText` has a string value:
```html
@if (loadingText) {
  <p>{{ loadingText }}</p>
} 
```
Also there's no need for the `class="loading-text"` in the html tag.